graylog2-web-interface/src/views/components/__snapshots__/WidgetQueryControls.test.jsx.snap

```diff
@@ -182,7 +182,7 @@ exports[`WidgetQueryControls should do something 1`] = `
         class="pull-right search-help"
       >
         <a
-          href="http://docs.graylog.org/en/3.2/pages/queries.html"
+          href="http://docs.graylog.org/en/1.0/pages/queries.html"
```

Comment:
Could we use something more obviously mocked than `1.0`?

Reply:
Also, let's adjust the path to use `https` and not `http`.

Reply:
Changed it to more obvious mock content.
I will create another PR for the https change, I know it's small change and I will have to rebase the PR, but I just don't want to mix the changes.